Images/descriptionPrompt.talon-list

```diff
@@ -0,0 +1,8 @@
+list:  user.descriptionPrompt
+
+-
+
+structure: Output nothing but your best approximation for the raw semantic HTML without any styling that could be used to create the user interface in the following image. Do not include any boilerplate HTML.
```

Comment:
I love this 😀

Reply:
I'd also like to see a prompt to generate some reasonable component names (I.e. if you were to build this in react, how might you break it up?)